lib/base/gff.ml

```diff
@@ -1,6 +1,6 @@
 (* https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md *)
 
-open Base
+
 open Printf
```

Comment:
I'd do `-open Printf` in the jbuild file. Core indirectly does this but for some reason Base doesn't.